greedy_methods/fractional_knapsack.py

```diff
@@ -11,21 +11,21 @@ def frac_knapsack(vl, wt, w, n):
     >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 8, 4)
     95.0
     >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6], 8, 4)
-    ?
+    60.0
     >>> frac_knapsack([10, 40, 30], [5, 4, 6, 3], 8, 4)
-    ?
+    60.0
     >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 0, 4)
-    ?
+    0
     >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 8, 0)
-    ?
+    95.0
     >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], -8, 4)
-    ?
+    0
     >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 8, -4)
-    ?
+    95.0
     >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 800, 4)
-    ?
+    130
     >>> frac_knapsack([10, 40, 30, 50], [5, 4, 6, 3], 8, 400)
-    ?
+    95.0
```

Comment:
Just one more... I promise.
```suggestion
    95.0
    >>> frac_knapsack("ABCD", [5, 4, 6, 3], 8, 400)
    ?
```